src/main/java/com/team1/dealerApp/models/TvShowUpdater.java

```diff
@@ -0,0 +1,23 @@
+package com.team1.dealerApp.models;
+
+import com.team1.dealerApp.entities.TvShow;
+import com.team1.dealerApp.mappers.TvShowMapper;
+import com.team1.dealerApp.repositories.TvShowRepository;
+import lombok.RequiredArgsConstructor;
+
+import java.lang.reflect.Field;
+
+@SuppressWarnings("unused")
+@RequiredArgsConstructor
+public class TvShowUpdater <T>{
```

Comment:
Mi piace molto questa soluzione con le generics per rintracciare il field, penso che la utilizzerò anche in Movie!